inspire_dojson/experiments/rules.py

```diff
@@ -108,11 +108,74 @@ def long_name(self, key, value):
 @experiments.over('inspire_classification', '^372..')
 @utils.for_each_value
 def inspire_classification(self, key, value):
-    schema = load_schema('experiments')
-    pattern = schema['properties']['inspire_classification']['items']['pattern']
-
-    if re.match(pattern, value.get('a')):
-        return value.get('a')
+    def _get_category(value):
+        EXPERIMENT_CATEGORIES_MAP = \
```

Comment:
you should pull this out and make it global, otherwise you need to rebuild this all the time.

Reply:
Ok, but should I leave it in experiment/rules.py or should I move it to utils/ for example?

Reply:
you can leave it here